catalyst_voices/packages/catalyst_voices_services/lib/src/storage/storage_string_mixin.dart

```diff
@@ -0,0 +1,76 @@
+//ignore_for_file: avoid_positional_boolean_parameters
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:catalyst_voices_services/src/storage/storage.dart';
+import 'package:catalyst_voices_services/src/storage/vault/secure_storage_vault.dart';
+import 'package:flutter/foundation.dart';
+
+/// Utility mixin which implements all but String read/write of [Storage]
+/// interface. Every method is has its mapping to [readString]/[writeString].
+///
+/// See [SecureStorageVault] as example.
+mixin StorageStringMixin implements Storage {
```

Comment:
StringStorageMixin? StorageAsStringMixin? Somehow current name sounds off

Reply:
Agree, will think about it